classes.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class DbConnect
+{
+    private string $host = "127.0.0.1:3306";
+    private string $user = "root";
+    private string $pwd = "password";
+    private string $dbName = "collectorapp";
```

Comment:
these should either be made into class constants, or make it so that you can set them differently for each new instance of DbConnect